app/src/main/java/pl/futurecollars/invoicing/controllers/InvoiceController.java

```diff
@@ -25,27 +27,37 @@
 
     @PostMapping
     public ResponseEntity<Invoice> save(@RequestBody Invoice invoice) {
+        invoice.generatedId();
+        log.debug("Cannot adding new invoice");
```

Comment:
shouldn't it be log.debag("Adding new invoice") ?

Reply:
Yes, it will probably look better.